index.js

```diff
@@ -149,24 +149,45 @@ class ServerlessCustomDomain {
       let nameLength = 0;
       // The arn of the choosen certificate
       let certificateArn;
+      // The certificate name
+      let certificateName = this.serverless.service.custom.customDomain.certificateName;
 
-      data.CertificateSummaryList.forEach((certificate) => {
-        let certificateListName = certificate.DomainName;
 
-        // Looks for wild card and takes it out when checking
-        if (certificateListName[0] === '*') {
-          certificateListName = certificateListName.substr(1);
+      // Checks if a certificate name is given
+      if (certificateName != null) {
+        if (certificateName.trim() === '') {
+          throw Error('certificateName is empty. Please put a valid certificate name or remove certificateName from the serverless file.');
```

Comment:
Not sure if this error is really needed. Could we just let it fail on its own with the `Could not find the certificate ${certificateName}` error?